Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.Properties.cs

```diff
@@ -103,6 +104,24 @@ public partial class MasterDetailsView
             new PropertyMetadata(null));
 
         /// <summary>
+        /// The detail template selector property
+        /// </summary>
+        public static readonly DependencyProperty DetailTemplateSelectorProperty = DependencyProperty.Register(
+            nameof(DetailTemplateSelector), typeof(DataTemplateSelector), typeof(MasterDetailsView), new PropertyMetadata(default(DataTemplateSelector)));
+
+        /// <summary>
+        /// Gets or sets the detail template selector.
+        /// </summary>
+        /// <value>
+        /// The detail template selector.
+        /// </value>
+        public DataTemplateSelector DetailTemplateSelector
```

Comment:
Should this be Detail**s**TemplateSelector as there is already the Detail**s**Template?